.github/workflows/cron-update-turbo.yml

```diff
@@ -0,0 +1,27 @@
+name: Cron Update Turbo
+
+on:
+  # Run every week https://crontab.guru/every-week
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        # 0 means fetch all commits so we can commit and push in the script below
+        with:
+          fetch-depth: 0
+      - name: install pnpm@7.26.0
+        run: npm i -g pnpm@7.26.0
```

Comment:
is `npm` available in actions without setting up node first?

Reply:
Not that I'm aware of. I copied this from https://github.com/vercel/vercel/blob/main/.github/workflows/test-unit.yml#L46-L47.